src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+import duke.command.Command;
+import duke.task.TaskList;
+
+public class Duke {
+    private final Storage storage;
+    private TaskList tasks;
+    private final Ui ui;
+
+    public Duke(String pathname) {
+        ui = new Ui();
+        storage = new Storage(pathname);
```

Comment:
should pathname be pathName instead?

Reply:
Pathname is a single word so I chose to use pathname instead of pathName. Here is Java's own documentation for their File class for reference: [https://docs.oracle.com/javase/8/docs/api/java/io/File.html](https://docs.oracle.com/javase/8/docs/api/java/io/File.html )